4.Buildroot/README.md

```diff
@@ -0,0 +1,77 @@
+<h2> How To Using Buildroot For Generate U-boot & RFS </h2>
+
+### Step 1: Download source and extract file
+I'm using version buildroot-2017.05
+```shell
+https://buildroot.org/downloads/
+```
+```shell
+tar xfv buildroot-2017.05.tar.gz
+```
```

Comment:
Can we add more info about buildroot?

Reply:
This information along with image files are enough